### ISSUE TITLE
dev: renaming kakarot precompiles related variables 

### ISSUE THREAD

**Issue by obatirou** [kkrt-labs/kakarot#1337]
Following https://github.com/kkrt-labs/kakarot/issues/1314, a renaming of function and related variables should be done

**Comment by ClementWalter** [comment]
is this still true?

**Comment by obatirou** [comment]
> is this still true?

Yes I think so, naming of variables could be improved to really grasp the intention 

https://github.com/kkrt-labs/kakarot/issues/1314#issuecomment-2275503554
> Kakarot_authorized_message_senders should only be the L1 address of the L1KakarotMessaging contract, that pre-formats messages to be compatible with Kakarot.
Kakarot_authorized_cairo_precompiles_callers refers to L2 EVM addresses that are authorized to call Cairo contracts. We need to whitelist them, as we cannot trust them to run non-failing transactions.
